src/kd/ctrl/ctrl.c

```diff
@@ -2152,11 +2162,11 @@ static s32 _sceCtrlReadBuf(SceCtrlData2 *pData, u8 nBufs, u32 port, u8 mode)
             * is an extended data buffer and fill it accordingly.
             */
            if (mode >= PEEK_BUFFER_POSITIVE_EXTRA) {
-			   pData->rsrv[0] = 0;
```

Comment:
This change here is purely cosmetic and was done for readability.

Reply:
The original ASM code has that code moved outside the if/else block and personally I prefer that over the cosmetic change done here. It's not critical to block this PR though.